src/picosvg/geometric_types.py

```diff
@@ -50,6 +50,11 @@ def __add__(self, other: "Vector") -> "Point":
     def round(self, digits: int) -> "Point":
         return Point(round(self.x, digits), round(self.y, digits))
 
+    def tolerent_equals(
```

Comment:
spelling

```suggestion
    def tolerant_equals(
```

I would have named this method `almost_equals`

Reply:
almost_equals sgtm